corehq/messaging/scheduling/forms.py

```diff
@@ -212,16 +212,24 @@ def set_app_and_form_unique_id_choices(self):
         self.fields['app_and_form_unique_id'].choices = [('', '')] + self.schedule_form.form_choices
 
     def clean_subject(self):
-        if self.schedule_form.cleaned_data.get('content') not in (ScheduleForm.CONTENT_EMAIL,
-                                                                  ScheduleForm.CONTENT_FCM_NOTIFICATION):
+        if (self.schedule_form.cleaned_data.get('content') == ScheduleForm.CONTENT_FCM_NOTIFICATION
+                and self.cleaned_data['fcm_message_type'] == FCMNotificationContent.MESSAGE_TYPE_NOTIFICATION):
+            cleaned_value = self._clean_message_field('subject')
+            return self._validate_fcm_message_length(cleaned_value, self.FCM_SUBJECT_MAX_LENGTH)
+
+        if self.schedule_form.cleaned_data.get('content') not in ScheduleForm.CONTENT_EMAIL:
```

Comment:
@ajeety4 I think this needs to be equal and not in.

Reply:
Resolved in 886681405033d284c46c19aa52298d6834b5f8e3.  Should have been `!=`